roles/manage-users/tasks/create_user.yml

```diff
@@ -10,6 +10,30 @@
       name: "{{ item.user_name | trim }}"
       mail: "{{ item.email | default('') }}"
 # The addition of expiration date is a request that will be submitted upstream
-#      expiration_date: "{{ item.expiration_date | default('') }}"
+#      expiration_date: "{{ this_user.expiration_date | default('') }}"
     with_items: "{{ users }}"
+    register: the_user
+
+  - name: "Display results of ipa_user loop"
+    debug:
+      var: item.user
+      verbosity: 2
+    with_items: "{{ the_user.results }}"
+
+  - name: "clear users"
```

Comment:
Maybe make this a bit more descriptive - i.e.: "Clear users before re-building list with additional data".